antora-ui-camel/src/css/frontpage.css

```diff
@@ -244,18 +222,120 @@ section.frontpage.columns .box .feature-icon {
   }
 
   .split img {
-    max-width: 70%;
     display: block;
     margin: auto;
   }
 
-  section.frontpage.columns .box {
+  .frontpage-header .main-heading,
+  section.frontpage.columns.functionalities,
+  section.frontpage.columns.projects,
+  section.frontpage.apache,
+  section.frontpage.blog-posts .blog-post {
+    width: 90%;
+    margin: 0 auto;
+  }
+
+  section.frontpage.columns.blog-posts {
+    flex-direction: column;
+  }
+
+  section.frontpage.blog-posts .blog-post .date,
+  section.frontpage.projects .project .icon {
+    min-width: 25%;
+  }
+
+  section.frontpage.projects .project,
+  section.frontpage.columns.functionalities .box {
+    width: 100%;
+  }
+
+  section.frontpage.functionalities .icon img {
+    max-width: 25vw;
+    margin: 2vw;
+  }
+
+  section.frontpage.functionalities .content {
+    width: 70vw;
+  }
+
+  .frontpage-header p {
+    font-size: inherit;
+    text-align: center;
+  }
+
+  .frontpage-header {
+    flex-direction: column-reverse;
```

Comment:
Perhaps:
```suggestion
    flex-direction: column;
```

To make the "Apache Camel" above the image, since currently image doesn't really depict what Camel is?

Reply:
> Perhaps:
> 
> To make the "Apache Camel" above the image, since currently image doesn't really depict what Camel is?

I completely understand your point, but between 1 & 2, I think 2 looks aesthetically better than 1. What do you think?

![image](https://user-images.githubusercontent.com/32356795/89732937-ab657700-da6f-11ea-81c5-f7705d0cef6e.png)  ![image](https://user-images.githubusercontent.com/32356795/89733016-3fcfd980-da70-11ea-9108-75fca6ee7bb0.png)